hbase-server/src/main/java/org/apache/hadoop/hbase/master/ActiveMasterManager.java

```diff
@@ -106,6 +114,31 @@ void handle(final String path) {
     }
   }
 
+  /*
+   * Fetches the active master's ServerName from zookeeper.
+   */
+  private void fetchAndSetActiveMasterServerName() {
+    LOG.debug("Attempting to fetch active master sn from zk");
```

Comment:
we can guard this with LOG.isDebugEnabled()

Reply:
That check is not needed on the caller side. Logger does it in it's call stack.